app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/services/HealthcheckService.scala

```diff
@@ -62,11 +87,12 @@ class HealthcheckService(xa: Transactor[IO])(implicit timer: Timer[IO],
   val routes: HttpRoutes[IO] =
     HttpRoutes.of {
       case GET -> Root =>
-        val healthcheck = (dbHealth |+| cacheHealth)
+        val healthcheck = (dbHealth |+| cacheHealth |+| gdalHealth)
           .through(mods.recoverToSick)
         healthcheck.check flatMap { check =>
           if (check.value.reduce.isHealthy) Ok("A-ok")
           else {
+            logger.warn("Healthcheck Failed (postres, memcached, or gdal)")
```

Comment:
Added logging so that we could search for this in our logs if we needed to at a later date